kernel/config.txt

```diff
@@ -21,12 +21,112 @@ necessary. Information about pre-built hardware as a whole will not be covered.
++------------------------------------------------------------------------------+
+
+which will land you with a premade, fairly generic and compatible configuration
+for your architecture.
+
+There are countless flows of configuring the Linux kernel, and there are fun
+things to try scattered all over the internet. Good luck!
+
+
+[2.0] Modifying a config
+________________________________________________________________________________
+
+There are several ways to modify an existing config file that are more
+convenient than editing each of them by hand, most of them very well
+documented. They arrange all of the config options in neat menus and submenus
+and provide descriptions for each of them, allowing some sort of form of the
+Linux kernel to be comprehended by mere mortals.
+
+The first method is:
+
++------------------------------------------------------------------------------+
+|                                                                              |
+|   $ make menuconfig                                                          |
+|                                                                              |
++------------------------------------------------------------------------------+
+
+which compiles and launches a terminal program in nostalgic colors, and allows
+you to browse the existing configuration, compartmentalised in trees and
+subtrees, using your keyboard and search for and change the configurations you
+want.
+
+Another terminal tool is:
+
++------------------------------------------------------------------------------+
+|                                                                              |
+|   $ make nconfig                                                             |
+|                                                                              |
++------------------------------------------------------------------------------+
+
+which is pretty similar, but different controls and in largely forgiving color
+schemes.
+
+Some GUI configuration tools are:
+
++------------------------------------------------------------------------------+
+|                                                                              |
+|   $ make xconfig                                                             |
+|                                                                              |
++------------------------------------------------------------------------------+
+
+and
+
++------------------------------------------------------------------------------+
+|                                                                              |
+|   $ make gconfig                                                             |
+|                                                                              |
++------------------------------------------------------------------------------+
+
+which require the QT and GTK framworks respectively, and of course, a graphical
+X server running.
```

Comment:
I think it's best to omit this portion as the user will not have GUI capabilities during the installation guide. It might be better to format it like this:

```
+------------------------------------------------------------------------------+
| Terminal based configuration tools (requires ncurses)                        |
+------------------------------------------------------------------------------+
|                                                                              |
|   $ make menuconfig                                                          |
|   $ make nconfig                                                             |
|                                                                              |
+------------------------------------------------------------------------------+
| Graphical configuration tools (requires a working Xorg server and QT/GTK)    |
+------------------------------------------------------------------------------+
|                                                                              |
|   $ make xconfig  # Requires qt5.                                            |
|   $ make gconfig  # Requires gtk+3.                                          |
|                                                                              |
+------------------------------------------------------------------------------+
```